recipe/meta.yaml

```
@@ -1,70 +1,67 @@
-{% set version = "0.12.0" %}
-{% set cov_fail_under = "82" %}
-{% set min_sphinx = "4.2" %}
-
+{% set name = "pydata-sphinx-theme" %}
+{% set version = "0.16.1" %}
 package:
   name: pydata-sphinx-theme
   version: {{ version }}
 
 source:
-  - url: https://pypi.io/packages/py3/p/pydata-sphinx-theme/pydata_sphinx_theme-{{ version }}-py3-none-any.whl
-    sha256: c17dbab67a3774f06f34f6378e896fcd0668cc8b5da1c1ba017e65cf1df0af58
-  - folder: src
-    url: https://github.com/pydata/pydata-sphinx-theme/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: 1759e12524b440d136228086206fbd23c3cf8007a64d3cecd7eda492f85617d3
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydata_sphinx_theme-{{ version }}.tar.gz
+  sha256: a08b7f0b7f70387219dc659bff0893a7554d5eb39b59d3b8ef37b8401b7642d7
+  patches:
+    - 000_set_nodejs_version.patch   # [linux]
 
 build:
   number: 0
-  noarch: python
+  skip: true   #[py<39 or s390x]
   script:
-    - {{ PYTHON }} -m pip install --find-links . --no-index --ignore-installed --no-deps pydata-sphinx-theme
-    - cd {{ RECIPE_DIR }} && {{ PYTHON }} collect_licenses.py
-
+    - export STB_USE_SYSTEM_NODE=1  # [linux]
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 requirements:
+  build:
+    - patch                                                         # [not win]
+    - m2-patch                                                      # [win]
```

```suggestion
    - patch  # [linux]
```
Only need patch installed on platforms using patches.